yarn.lock

```diff
@@ -1746,9 +1746,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001043, caniuse-lite@^1.0.30001061:
```

Comment:
I think this file is not from this PR. WDYT?

Reply:
I separate it for another PR